sparqlQueries/listCachedFdps.sparql

```diff
@@ -0,0 +1,7 @@
+PREFIX r3d: <http://www.re3data.org/schema/3-0#>
+PREFIX dct: <http://purl.org/dc/terms/>
+
+SELECT DISTINCT ?fdp ?name {
+    ?fdp r3d:dataCatalog ?catalog ;
```

Comment:
Maybe you can get the type of fdpURI right? This statement returns empty fdp URI if the repository doesn't have any catalogies.

Reply:
Good point, I blindly followed the example you gave me, but that fails in your scenario.